.circleci/config.yml

```diff
@@ -10,13 +10,11 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Docker-compose validate
```

Comment:
here and below - the config is identical to the CI config in `govcms8lagoon` project.